scripts/west_commands/runners/__init__.py

```diff
@@ -60,6 +60,7 @@ def _import_runner_module(runner_name):
     'trace32',
     'uf2',
     'xtensa',
+    'xsdb',
```

Comment:
Please read the line below this one ;)

Reply:
Since you're touching this file, wrap these entries with:

```
# zephyr-keep-sorted-start
...
# zephyr-keep-sorted-stop
```

Reply:
Sure will fix...

Reply:
It appears the list wasn't sorted to begin with, I've created #80061 to fix this.